Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1102,6 +1102,10 @@ class BrowserViewController: UIViewController {
                 webView.evaluateJavaScript("\(ReaderModeNamespace).checkReadability()", completionHandler: nil)
             }
 
+            if urlBar.inOverlayMode, InternalURL.isValid(url: url), url.path.starts(with: "/about/home") {
```

Comment:
There is a constant for this: `AboutHomeHandler.path`, but you would still need to add a slash at the beginning of the path